### PR TITLE
Add libpixman

### PR DESCRIPTION
**Pull request by CRKatri**

### CODE REVIEW

File: libpixman.mk [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	touch $(BUILD_WORK)/libpixman/.build_complete
+endif
+
+libpixman-package: libpixman-stage
+	# libpixman.mk Package Structure
+	rm -rf $(BUILD_DIST)/libpixman{-dev}
+	mkdir -p $(BUILD_DIST)/libpixman/usr/lib \
+		$(BUILD_DIST)/libpixman-dev/usr/{lib,include/pixman-1}
+	
+	# libpixman.mk Prep libpixman
```
Comment (Diatrus) [inline libpixman.mk:36]: Please keep debian naming scheme. Splitting isn't just about splitting, it's also about having the so_versions in the lib package name.

Comment (CRKatri) [reply libpixman.mk:36]: So package them as `libpixman-1-0*`?

Comment (Diatrus) [reply libpixman.mk:36]: Exactly what Debian names it.

File: libpixman.mk [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+endif
+
+libpixman-package: libpixman-stage
+	# libpixman.mk Package Structure
+	rm -rf $(BUILD_DIST)/libpixman-1-0{-dev}
+	mkdir -p $(BUILD_DIST)/libpixman-1-0/usr/lib \
+		$(BUILD_DIST)/libpixman-1-dev/usr/{lib,include/pixman-1}
+	
+	# libpixman.mk Prep libpixman
+	cp -a $(BUILD_STAGE)/libpixman/usr/lib/libpixman-1.0.40.0.dylib $(BUILD_DIST)/libpixman-1-0/usr/lib
```
Comment (Diatrus) [inline libpixman.mk:39]: Change `libpixman-1.0.40.0.dylib` to `libpixman-1.0*.dylib

File: build_info/libpixman-1-0.control [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Package: libpixman-1-0
+Version: @DEB_LIBPIXMAN_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Section: Multimedia
```
Comment (Diatrus) [inline build_info/libpixman-1-0.control:5]: This should be Libraries section

File: libpixman.mk [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+libpixman-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://cairographics.org/releases/pixman-$(LIBPIXMAN_VERSION).tar.gz
+	$(call EXTRACT_TAR,pixman-$(LIBPIXMAN_VERSION).tar.gz,pixman-$(LIBPIXMAN_VERSION),libpixman)
+
+ifneq ($(wildcard $(BUILD_WORK)/libpixman/.build_complete),)
+libpixman:
+	@echo "Using previously built libpixman."
+else
+libpixman: libpixman-setup libpng16
```
Comment (Diatrus) [inline libpixman.mk:17]: As far as I can tell libpng16 is not a build dependency.

Comment (CRKatri) [reply libpixman.mk:17]: Just double-checked, and can confirm that is a build dependency.

**Pull request merged**